consensus/polybft/block_builder.go

```diff
@@ -194,17 +195,8 @@ func (b *BlockBuilder) writeTxPoolTransaction(tx *types.Transaction) (bool, erro
 		return true, nil
 	}
 
-	if tx.ExceedsBlockGasLimit(b.params.GasLimit) {
```

Comment:
Don't we need this check?

Reply:
It's inside `WriteTx`.

Reply:
It's not that easy to see (I didn't also) but the code really works. Maybe we need some comment to explain why/how it is working?

Reply:
Any suggestion how such comment should look like? It isn't meaningful to me adding comment where I'm explaining that given check is performed in the `WriteTx` function instead of this one :smile:

Reply:
I do not know :D Maybe we do not need it at all. @b00ris wdyt?